src/ScottPlot5/ScottPlot5 Sandbox/Sandbox.UnoPlatform/Sandbox.UnoPlatform/Properties/launchSettings.json

```diff
@@ -0,0 +1,50 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:8080",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    // This profile is first in order for dotnet run to pick it up by default
+    "Sandbox.Unplatform (WebAssembly)": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "http://localhost:5000",
+      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "Sandbox.Unplatform (WebAssembly IIS Express)": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    // Note: In order to select this profile, you'll need to comment the `Packaged` profile below until this is fixed: https://aka.platform.uno/wasdk-maui-debug-profile-issue
+    "Sandbox.Unplatform (WinAppSDK Unpackaged)": {
+      "commandName": "Project",
+      "compatibleTargetFramework": "windows"
+    },
+    "Sandbox.Unplatform (WinAppSDK Packaged)": {
+      "commandName": "MsixPackage",
+      "compatibleTargetFramework": "windows"
+    },
+    "Sandbox.Unplatform (Desktop)": {
+      "commandName": "Project",
+      "compatibleTargetFramework": "desktop"
+    },
+    "Sandbox.Unplatform (Desktop WSL2)": {
```

Comment:
Jerome I am currently unable to launch Desktop WSL2 with the Sandbox because of space in the path 
(related previous details: https://github.com/unoplatform/private/issues/479#issuecomment-2221864398)
Since I don't think we will change the entire folder naming here, should I just leave a comment in MainPage.xaml regarding that ? Or wait until it is time to review the changes with Scott to mention it to him?
![image](https://github.com/user-attachments/assets/3cce089c-721f-453c-9bf4-5e962833d05b)